### ISSUE TITLE
你好，请问什么时候公开代码

### ISSUE THREAD

**Issue by Guohaitao1** [jkwang28/OSDFace#7]
很棒的工作，期待尽快公开代码，或者预计何时公开呢，期待您的恢复，谢谢！

**Comment by ArenFrank** [comment]
坐等code update

**Comment by dongdk** [comment]
or can u release the EA-DISTS loss part?

**Comment by cnxft-s** [comment]
请问可以更新一下判别器部分的代码吗？

**Comment by jkwang28** [comment]
感谢您的理解。我们终于通过了校企合作项目的验收，现在已经公布了预训练模型和测试代码。感谢您的耐心等待，为您带来不便烦请谅解。 关于训练代码，我们已向公司申请，会在申请同意后，第一时间公布。

再次感谢您的理解！

Thank you for your understanding. We have finally passed the acceptance of the industry-university collaboration project, and the pre-trained model and test code have now been released. We appreciate your patience and apologize for any inconvenience caused.

Regarding the training code, we have applied to the company and will release it as soon as we receive approval.

Once again, thank you for your understanding.
